migrations.go

```diff
@@ -14,13 +12,11 @@ import (
 
 func CheckMigration() error {
 	stump.Log("checking if repo migration is needed...")
-	p := util.IpfsDir()
-	oldverB, err := ioutil.ReadFile(filepath.Join(p, "version"))
+	oldver, err := GetCurrentVersion()
```

Comment:
I don't know if this will work right. The version returned here is the ipfs binary version, and we're looking for the version of the repo.